src/store/elementalChat.js

```diff
@@ -261,9 +261,9 @@ export default {
     handleMessageSignal: ({ commit }, payload) => {
       log('adding signal message: ', payload)
       commit('addChannels', [payload.channelData])
-      commit('addMessage', {
-        channelId: payload.channelData.entry.uuid,
-        message: payload.messageData
+      commit('addMessagesToChannel', {
+        channel: payload.channelData,
+        messages: [payload.messageData]
```

Comment:
This (and the removal of `addMessage` below) are reverting changes that fixed a bug with signals. I think we want to keep those changes.